src/services/ui/src/pages/detail/index.tsx

```diff
@@ -59,9 +59,10 @@ const StatusCard = (data: opensearch.main.Document) => {
           </em>
         )}
         {user?.isCms && checker.isInSecondClock && (
-          <span id="secondclock" className="ml-2">
-            2nd Clock
-          </span>
+          <>
+            <br />
```

Comment:
Let's sync up today and go over some better methods of handling these layouts without breakpoints. These elements are meant for breaking up text in a single element, not between multiple elements. We can achieve this result with proper styling and no breakpoint elements

Reply:
@kevinhaube This is not a big thing, i have made the changes.